cmd/utils/flags.go

```diff
@@ -1453,119 +1414,15 @@ func setIstanbul(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	if ctx.GlobalIsSet(LegacyIstanbulProposerPolicyFlag.Name) {
 		log.Warn("Flag value is ignored, and obtained from genesis config", "flag", LegacyIstanbulProposerPolicyFlag.Name)
 	}
-	cfg.Istanbul.ReplicaStateDBPath = stack.ResolvePath(cfg.Istanbul.ReplicaStateDBPath)
```

Comment:
A good "clean up" would be to move all CELO specific flag code to a celo_flags.go file